MIGRATION.md

```diff
@@ -863,6 +864,21 @@ Starting in 7.0 the `grid.cellSize` parameter should now be `backgrounds.grid.ce
 
 We removed the deprecated `withA11y` decorator. This was [deprecated in 6.0](#removed-witha11y-decorator)
 
+#### Addon-interactions: Interactions debugger is now default
+
+The interactions debugger in the panel is now displayed by default. The feature flag is now removed.
+
+```js
+// .storybook/main.js
+
+const config = {
+  features: {
+    interactionsDebugger: true, // This should be removed!
```

Comment:
```suggestion
    interactionsDebugger: true, // This will be ignored
```

Reply:
@JReinhold I don't agree.. the instruction here to have users remove it... not to indicate that it will be ignored.